src/Sylius/Behat/Page/Admin/Customer/UpdatePage.php

```diff
@@ -0,0 +1,55 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sylius\Behat\Page\Admin\Customer;
+
+use Sylius\Behat\Page\Admin\Crud\UpdatePage as BaseUpdatePage;
+
+/**
+ * @author Anna Walasek <anna.walasek@lakion.com>
+ */
+class UpdatePage extends BaseUpdatePage implements UpdatePageInterface
+{
+    /**
+     * @var array
+     */
+    protected $elements = [
+        'email' => '#sylius_customer_email',
+        'gender' => '#sylius_customer_gender',
+        'birthday' => '#sylius_customer_birthday',
+    ];
+
+    /**
+     * {@inheritdoc}
+     */
+    public function assertElementHasValue($elementName, $elementValue)
```

Comment:
Assertion usually throws an exception on failure. Anyway, we should assert in the context based on the data retrieved from the page.